tests/unit/utils/network_test.py

```diff
@@ -106,8 +106,13 @@ def test_sanitize_host(self):
         self.assertEqual(ret, '10.1.2.3')
 
     def test_host_to_ips(self):
+        '''
+        NOTE: When this test fails it's usually because the IP address has
+        changed. In these cases, we just need to update the IP address in the
+        assertion.
+        '''
         ret = network.host_to_ips('www.saltstack.com')
-        self.assertEqual(ret, ['104.199.122.13'])
+        self.assertEqual(ret, ['104.197.168.128'])
```

Comment:
Yes, long-term this should be mocked. However, we're trying to cut the RC and the focus was getting the tests passing in the short term.